client/dashboard/sites/features.ts

```diff
@@ -1,52 +1,95 @@
 import { DotcomFeatures } from '../data/constants';
-import {
-	hasPlanFeature,
-	hasHostingFeatures,
-	hasAdvancedHostingFeatures,
-} from '../utils/site-features';
+import { hasPlanFeature } from '../utils/site-features';
 import type { Site, User } from '../data/types';
 
+export const HostingFeatures = {
+	SETTINGS_PHP: DotcomFeatures.SFTP,
+	SETTINGS_SFTP: DotcomFeatures.SFTP,
+	SETTINGS_SSH: DotcomFeatures.SSH,
+	SETTINGS_DATABASE: DotcomFeatures.SFTP,
+	SETTINGS_PRIMARY_DATA_CENTER: DotcomFeatures.SFTP,
+	SETTINGS_STATIC_FILE_404: DotcomFeatures.SFTP,
+	SETTINGS_CACHING: DotcomFeatures.ATOMIC,
+	SETTINGS_DEFENSIVE_MODE: DotcomFeatures.SFTP,
+	SETTINGS_RESTORE_PLAN_SOFTWARE: DotcomFeatures.ATOMIC,
+} as const;
+
+export type HostingFeatures = ( typeof HostingFeatures )[ keyof typeof HostingFeatures ];
+
+export function hasHostingFeature( site: Site, feature: HostingFeatures ) {
+	return (
+		site.is_wpcom_atomic &&
+		! site.plan?.expired &&
+		hasPlanFeature( site, feature as DotcomFeatures )
```

Comment:
Nit: We can change the parameter type as below and then we don't need the conversion

```ts
hasPlanFeature( site: Site, feature: `${ DotcomFeatures }` ) {}
```